google-oauth-client/src/main/java/com/google/api/client/auth/oauth/OAuthHmacSigner.java

```diff
@@ -32,14 +34,34 @@
 @Beta
 public final class OAuthHmacSigner implements OAuthSigner {
 
+  private final Map<String, String> signatureMethodMap = ImmutableMap.of(
+          "HMAC-SHA1", "HmacSHA1",
```

Comment:
I don't understand why you need both the keys and the values here.

Reply:
Good question.  The signature method that need to be used for the OAuth 1.0 header has to be of the format:  "HMAC-SHA1" or "HMAC-SHA2", while the actual Algorithm strings that Java requires have to be of the format "HmacSHA1" or "HmacSHA256" as described here:  https://docs.oracle.com/javase/9/docs/specs/security/standard-names.html#mac-algorithms.

My intent was to use a Map to associate the signature method as the key with the corresponding algorithm as the value, so there would not be an accidental mismatch.

Reply:
this has been reverted back in favor of a new class